Adding Swift support to documentation

TNuzzi opened the pull request:
I updated the documentation to use TLYShyNavBar component in a Swift project.

Mazyod commented:
Thanks a lot for this!

I just don't like how the TLYShyNavBar classes are copied to the swift demo. If you can please delete the copies and reference the original classes, that would be stellar.

TNuzzi commented:
Good point.  I removed the copied TLYShyNavBar folder and changed the project to reference the one included.  However I did update "TLYShyNavBarManager.h" and "TLYShyViewController.h" to include #import <UIKit/UIKit.h>.  As a result I updated the docs and removed the step to add the #import to the .h files.

Mazyod commented:
:star2:
